framework/src/headers/ip/v6/mod.rs

```diff
@@ -139,14 +156,14 @@ impl fmt::Display for Ipv6Header {
         let dst = Ipv6Addr::from(self.dst());
         write!(
             f,
-            "{} > {} version: {} traffic_class: {} flow_label: {} len: {} next_header: {} hop_limit: {}",
+            "{} > {} version: {} traffic_class: {} flow_label: {} len: {} next_header: {:?} hop_limit: {}",
             src,
             dst,
             self.version(),
             self.traffic_class(),
             self.flow_label(),
             self.payload_len(),
-            self.next_header(),
+            self.next_header().unwrap(),
```

Comment:
not safe. this could blow up.

Reply:
@drunkirishcoder think it's ok for now, but I'll wrap it up in my next PR around the test, cool?